templates/geosafe/analysis/create_base.html

```diff
@@ -526,21 +526,22 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
 
             check_popover();
 
-            var OpenMapSurfer_Roads = L.tileLayer('http://korona.geog.uni-heidelberg.de/tiles/roads/x={x}&y={y}&z={z}', {
-                zIndex: 0,
-                noWrap: true,
-                attribution: 'Imagery from <a href="http://giscience.uni-hd.de/">GIScience Research Group @ University of Heidelberg</a> &mdash; Map data &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
-            });
             var osm = L.tileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
                 zIndex: 0,
                 attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
             });
-            var base_maps = {
-                'OpenMapSurfer_Roads': OpenMapSurfer_Roads,
-                'OpenStreetMap': osm
-            };
+            var base_maps = {};
+            {% if LEAFLET_CONFIG and LEAFLET_CONFIG.TILES %}
+                {% for tile in LEAFLET_CONFIG.TILES %}
+
+                    base_maps["{{ tile.0 }}"] = L.tileLayer('{{ tile.1|safe }}', {
+                        attribution: '{{ tile.2|safe }}'
+                });
+                {% endfor %}
+            {% endif %}
```

Comment:
```suggestion
            default_basemap = base_maps["{{ LEAFLET_CONFIG.TILES.0 }}"];
            {% endif %}
```

Reply:
changes it into:
`default_basemap = base_maps["{{ LEAFLET_CONFIG.TILES.0.0 }}"];`